public/static/locales/en/analyses.json

```diff
@@ -1,85 +1,83 @@
 {
-    "all":"All",
-    "analyses":"Analyses",
-    "analysis":"Analysis",
-    "analysisId":"Analysis ID",
-    "analysisInfo":"View Analysis Info",
-    "analysisInfoDlgTitle":"Analysis Info",
-    "analysisInfoFetchError":"Unable to fetch analysis information. Please try again.",
-    "analysisInfoTitle":"Analysis Info",
-    "analysisParamFetchError":"Unable to fetch analysis parameters. Please try again.",
-    "analysisParamTitle":"Viewing parameters for {{name}}",
-    "analysisParams":"Parameters",
-    "analysisRenameError": "Unable to rename analysis. Please try again.",
```

Comment:
Looks like this `analysisRenameError` message got accidentally removed somehow.

Reply:
😖 . I might have missed it during rebase. Let me add it back.

Reply:
This should be fixed.